app.rb

```diff
@@ -146,9 +146,19 @@ def authorization_url(ial:, aal: nil)
         state: random_value,
         nonce: random_value,
         prompt: 'select_account',
-      }.to_query
+      }
```

Comment:
do we need an entire separate `user_options` need to be a separate method? seems like we could just add a little ternary and a `.compact to this hash and remove a ton of diff noise:

```
  prompt: 'select_account',
  inherited_proofing_auth: params['ip_auth_option'] ? random_value : nil
}.compact.to_query
```

Reply:
Diff noise silenced. :+1: